startup.py

```diff
@@ -21,6 +21,6 @@
     elif choice == '2':
         dbmanager.stop()
     elif choice == '3':
-        initdb.main("","")
+        initdb.main()
```

Comment:
@asclines what's the problem? I'm just now taking a look at this, but not sure what the issue is.